### DISCUSSION TITLE
希望南丁格尔玫瑰图的label的type支持spider

### DISCUSSION THREAD

**Discussion by mago960806** [antvis/G2Plot#2366] [Ideas]
南丁格尔玫瑰图的label属性里的type只有inner和outer，如果想实现饼图中的spider有什么办法吗？

**Comment by visiky** [comment]
目前还不支持。感兴趣的钉钉或微信私聊我，提供贡献开发思路

**Comment by visiky** [comment]
### 设计建议：

- [ ] 请用正经拉线 label （类似饼图) ✅ 目前用坐标轴线来替代展示
- [x] 扇区请从12点钟方向开始 ✅ 设置 startAngel:  -Math.PI / 8 * 3.999,

### 优化 1

https://codesandbox.io/s/rose-label-fhzvu

| Before | After |
| --- | --- |
|![image](https://user-images.githubusercontent.com/15646325/122337719-4a50de80-cf71-11eb-8605-e45cf903687a.png)|![image](https://user-images.githubusercontent.com/15646325/122337675-3e651c80-cf71-11eb-901b-6ec6f519df04.png)|

**Comment by visiky** [comment]
会在新版中支持
<img width="707" alt="image" src="https://user-images.githubusercontent.com/15646325/206857816-8b260e64-545f-4c6f-8bc1-5b07ce5ef15a.png">
